packages/openapi3/README.md

```diff
@@ -86,6 +86,15 @@ By default all types declared under the service namespace will be included. With
 If the generated openapi types should have the `x-typespec-name` extension set with the name of the TypeSpec type that created it.
 This extension is meant for debugging and should not be depended on.
 
+#### `safeint-strategy`
+
+**Type:** `"double-int" | "int64"`
+
+How to handle safeint type. Options are:
+
+- `double-int`: Will produce `type: integer, format: double-int`
+- `int64`: Will produce `type: integer, format: int64`
```

Comment:
It's a shame the default setting isn't generated here

Reply:
yeah but its something we could add to the ref doc emitter.